docs/sw-update-conf.json

```diff
@@ -92,6 +92,11 @@
           "repo": "MrBeamLedStrips",
           "pip": "https://github.com/mrbeam/MrBeamLedStrips/archive/{target_version}.zip",
           "global_pip_command": true,
+          "beamos_version": {
+            "0.18.0": {
+              "pip_command": "sudo /usr/local/mrbeam_ledstrips/venv/bin/pip"
+            }
+          },
           "beamos_date": {
```

Comment:
I still can see the date

Reply:
I only added the changes to the PR#7
https://github.com/mrbeam/beamos_config/pull/7